Makefile

```diff
@@ -118,19 +120,19 @@ images: satellite-image storagenode-image uplink-image gateway-image ## Build ga
 
 .PHONY: gateway-image
 gateway-image: ## Build gateway Docker image
-	${DOCKER_BUILD} --pull=true -t storjlabs/gateway:${TAG}${CUSTOMTAG} -f cmd/gateway/Dockerfile .
+	${DOCKER_BUILD} --pull=true -t storjlabs/gateway:${TAG}${CUSTOMTAG} -t storjlabs/gateway:${LATEST_TAG} -f cmd/gateway/Dockerfile .
```

Comment:
Does that work?

Reply:
Pushing two tags would but the additional tag wasn't needed as we conditionally tag it down below.